tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    Testing the backend for custom images (Adding ImageStream to redhat-ods-applications)
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
+Library          JupyterLibrary
+Library          OpenShiftCLI
+Suite Teardown   Custom Image Teardown
+
+
+*** Variables ***
+${YAML} =         configs/resources/custom_image.yaml
+
+
+*** Test Cases ***
+Verify ImageStream Can Be Created
+    [Documentation]    Applies the YAML and Gets the ImageStream
+    [Tags]    Sanity
+    ...       ODS-1208
+    ${apply_status} =    Run Keyword And Return Status    Run    oc apply -f ${YAML}
+    # OpenShiftCLI.Apply    kind=ImageStream    src="configs/resources/custom_image.yaml"
+    # ...    namespace=redhat-ods-applications
+    ${get_status} =    OpenShiftCLI.Get    kind=ImageStream    field_selector=metadata.name==custom-test-image
```

Comment:
this one will not return status. Could we change the variable name to get_data or get_payload something like this? Also I think it would be good to have Common keyword for status so that people can  also use in Future.

Reply:
I also  think you  should create a keyword `Apply Configuration` (or similar), probably in tests/Resources/OCP.resource

Reply:
For applying the yaml we already have a keyword, which is `OpenShiftCLI.Apply` (but it was giving me issues). I don't think we need another keyword to do the same thing to be honest.
Agree with Tarun that the variable can be renamed and we can probably write a keyword to get the status from the data that `OpenShiftCLI.Get` returns.